build-farm/platform-specific-configurations/linux.sh

```diff
@@ -78,7 +78,18 @@ fi
 
 if [ "${ARCHITECTURE}" == "arm" ]
 then
-  export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-jobs=4 --with-memory-size=2000"
+  if lscpu | grep aarch64; then
+     echo Validating 32-bit environment on 64-bit host - perhaps it is a docker container ...
```

Comment:
Can we detect if it is a Docker container to add clarity?

Reply:
I'm not sure there is significant value to detecting it explicitrly, and bear in mind it would also be true if you were running a 32-bit userspace on a 64-bit kernel (which is a valid configuratoin - I've used it on the raspberry pi OS before) - I could remove that comment, but the docker environment is where I intend to use it in our build environment.